help_to_heat/portal/email_handler.py

```diff
@@ -74,24 +77,33 @@ def _send_token_email(user, subject, template_name, from_address, url_name, toke
     url = str(furl.furl(url=base_url, path=url_path, query_params={"code": token, "user_id": str(user.id)}))
     context = dict(user=user, url=url, contact_address=settings.CONTACT_EMAIL)
     body = render_to_string(template_name, context)
-    response = send_mail(
-        subject=subject,
-        message=body,
-        from_email=from_address,
-        recipient_list=[user.email],
-    )
-    return response
+    try:
+        response = send_mail(
+            subject=subject,
+            message=body,
+            from_email=from_address,
+            recipient_list=[user.email],
+        )
+        return response
+    except Exception as err:
+        logger.error("An error occured while attempting to send an email.")
+        logger.error(err)
```

Comment:
Presumably this is just swallowing the error. Might we want to surface the error to whatever called this to handler, or are we confident that logging and throwing the attempt away is correct?